getting-started/dcr-guide.rst

```diff
@@ -0,0 +1,97 @@
+######################################
+Generating DCR Coadds with LSST ComCam
+######################################
+
+This article walks through the building of differential chromatic refraction (DCR) coadds by running the
+``dcrAssembleCoaddTask`` on an example set of ComCam image data. Prior to generating DCR coadds, image
+templates must already be imported.
+
+.. note::
+
+   This guide assumes the user has access to a shared Butler repository containing data from LSST ComCam via
+   the `US Data Facility (USDF) <https://developer.lsst.io/usdf/storage.html>`__. This guide further assumes
+   the user has a recently-built version of ``lsst.distrib`` from the `LSST Science Pipelines
+   <https://developer.lsst.io/usdf/stack.html>`__ (circa ``w_2025_09`` or later).
+
+Importing AP Templates
+=======================
+For detailed instructions on importing AP Templates, see the `importing good seeing templates
+<https://pipelines.lsst.io/getting-started/dc2-guide.html>`__ guide for details.
+
+
+Assembling DCR Coadds
+=====================
+To generate DCR coadds, use the ``dcrAssembleCoaddTask`` found in the ``drp_tasks`` package. This requires
+image templates. Note: Register the DCR subfilters need prior to creating DCR coadds by following the
+`register-dcr-subfilters guide <https://pipelines.lsst.io/modules/lsst.daf.butler/scripts/butler.html#butler-register-dcr-subfilters>`__.
+
+
+Creating Your DCR Assemble Coadd Pipeline
+-----------------------------------------
+The first step in generating DCR coadds is to make a pipeline that runs the ``dcrAssembleCoaddTask`` from the
+LSST ``drp_tasks`` package. A sample ``dcrAssembleCoadd`` pipeline is shown below.
+
+.. code-block:: yaml
+
+    description: A DCR assemble coadd pipeline for LSSTComCam.
+
```

Comment:
It would be helpful to define `instrument` in this example, since that is required